db/migrate/20180911160120_add_entry_type_to_submission_entries.rb

```diff
@@ -0,0 +1,6 @@
+class AddEntryTypeToSubmissionEntries < ActiveRecord::Migration[5.2]
+  def change
+    add_column :submission_entries, :entry_type, :string
+    add_index :submission_entries, :entry_type
```

Comment:
Are we indexing pre-emptively here or is there a specific query or scope that needs this now?

Reply:
At least two places we'll need to scope on this: the finance report, data warehouse export.